extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -455,11 +458,12 @@ export class AmpAnalytics extends AMP.BaseElement {
       const argList = match[2] || '';
       let raw = (event && event['vars'] && event['vars'][name]) ||
           (trigger['vars'] && trigger['vars'][name]) ||
-          (this.config_['vars'] && this.config_['vars'][name]);
+          (this.config_['vars'] && this.config_['vars'][name]) ||
+          '';
       if (typeof raw == 'string') {
         raw = this.expandTemplate_(raw, trigger, event, opt_iterations - 1);
```

Comment:
Is there a point to try to expand `raw` again if it's an empty string, since you fallback to it above?